src/screens/ErrorPage.js

```diff
@@ -0,0 +1,28 @@
+import { Button } from "../components";
+import { render } from "../shared/dom";
+
+function onClickHandler() {
+    this.$routes.push('Home');
+}
+
+const template = `
+<div id="error-page">
```

Comment:
You should stick to the BEM methodology in here

Reply:
ok